diff --git a/node_modules/collect-v8-coverage/index.js b/node_modules/collect-v8-coverage/index.js
index 08b7853..db9ea66 100644
--- a/node_modules/collect-v8-coverage/index.js
+++ b/node_modules/collect-v8-coverage/index.js
@@ -13,6 +13,8 @@ class CoverageInstrumenter {
   async startInstrumenting() {
     this.session.connect();
 
+    await this.postSession('Debugger.enable');
+
     await this.postSession('Profiler.enable');
 
     await this.postSession('Profiler.startPreciseCoverage', {
@@ -30,6 +32,8 @@ class CoverageInstrumenter {
 
     await this.postSession('Profiler.disable');
 
+    await this.postSession('Debugger.disable');
+
     // When using networked filesystems on Windows, v8 sometimes returns URLs
     // of the form file:////<host>/path. These URLs are not well understood
     // by NodeJS (see https://github.com/nodejs/node/issues/48530).
